### PR TITLE
fix: bug with iron-overlay-backdrop when on-screen keyboard is shown

### PR DESCRIPTION
**Pull request by jdbence**
Any reason why `vh` was used instead of `%`?
Should resolve issue #149

### CODE REVIEW

**Comment by valdrinkoshi** [comment]
Thanks for the contribution @jdbence 
Could you add a test so we avoid any future regression? The test should check that the backdrop bounds match the body bounds (make the body scrollable as you did in your example on the issue)

File: test/iron-overlay-backdrop.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+    <link rel="import" href="../../iron-test-helpers/iron-test-helpers.html">
+    <link rel="import" href="test-overlay.html">
+  	<style>
+  		html,
```
Comment (valdrinkoshi) [inline test/iron-overlay-backdrop.html]: Indentation wrong

File: test/iron-overlay-backdrop.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  	    body {
+  		  margin: 0;
+  		  width: 100%;
+  		  height: 100%;
+  		  min-width: 0;
+  		}
+  		.sizer {
+  			width: 4000px;
```
Comment (valdrinkoshi) [inline test/iron-overlay-backdrop.html]: Indentation wrong

File: test/iron-overlay-backdrop.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+    <link rel="import" href="../../iron-test-helpers/iron-test-helpers.html">
+    <link rel="import" href="test-overlay.html">
+  	<style>
+  		html,
+  	    body {
+  		  margin: 0;
+  		  width: 100%;
+  		  height: 100%;
```
Comment (valdrinkoshi) [inline test/iron-overlay-backdrop.html]: Is this really needed? We want body to be scrollable, so we should not set the height, right?

**Comment by valdrinkoshi** [comment]
Comments on the test, which needs to be formatted with the right indentation across.
Thanks for doing this @jdbence!

**Comment by valdrinkoshi** [comment]
Hi @jdbence, your test is not being run, you need to import it in test/index.html
Or just add that suite and test fixture into test/iron-overlay-behavior.html

**Comment by jdbence** [comment]
Hi @valdrinkoshi,  I've updated the index file to include the test suite.

File: test/index.html [hunk 4]
```diff
@@ -22,7 +22,9 @@
     <script>
       WCT.loadSuites([
         'iron-overlay-behavior.html',
-        'iron-overlay-behavior.html?dom=shadow'
+        'iron-overlay-behavior.html?dom=shadow',
+        'iron-overlay-backdrop.html',
+		'iron-overlay-backdrop.html?dom=shadow',
```
Comment (valdrinkoshi) [inline test/index.html]: (sorry but) indentation's wrong here :x

File: test/iron-overlay-backdrop.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
```
Comment (valdrinkoshi) [inline test/iron-overlay-backdrop.html]: (c) 2016

**Comment by valdrinkoshi** [comment]
@jdbence I left 2 comments..Almost there :)

File: test/index.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html><!--
 @license
-Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
```
Comment (valdrinkoshi) [inline test/index.html]: wrong file, this should remain 2015.

**Comment by valdrinkoshi** [comment]
@jdbence test/iron-overlay-backdrop.html should be @2016 as it was created now. test/index.html should stay @2015

**Comment by jdbence** [comment]
Got it. That makes more sense.

**Comment by valdrinkoshi** [comment]
LGTM :+1: 
Thanks for the contribution @jdbence! :tada:

**Pull request merged**